pontoon/translations/views.py

```diff
@@ -170,18 +195,27 @@ def create_translation(request):
                 description=desc,
             )
 
-    # Award Translation Champion Badge stats
-    if user.badges_translation_count in settings.BADGES_TRANSLATION_THRESHOLDS:
-        # TODO: Send a notification to the user
-        pass
-
-    return JsonResponse(
-        {
-            "status": True,
-            "translation": translation.serialize(),
-            "stats": TranslatedResource.objects.stats(project, paths, locale),
+    # Send Translation Champion Badge notification information
+    response_data = {
+        "status": True,
+        "translation": translation.serialize(),
+        "stats": TranslatedResource.objects.stats(project, paths, locale),
+    }
+
```

Comment:
You could factor out the code below and call it like this:

```python
_add_badge_data(response_data, request.user, badge_name)
```

And the code would look something along the lines of:
```python
def _add_badge_data(response_data, user, badge_name):
    count = user.badges_review_count
    level = settings.BADGES_TRANSLATION_THRESHOLDS.index(count) + 1

    if count in settings.BADGES_TRANSLATION_THRESHOLDS:
        response_data["badge_update"] = {
            "name": badge_name,
            "level": level,
        }
        _send_badge_notification(
            user,
            badge_name,
            level,
        )
```